src/common/MerjoonTransformer.ts

```diff
@@ -37,6 +37,20 @@ export class MerjoonTransformer implements IMerjoonTransformer {
           case 'STRING':
             newVal = value?.[key].toString();
             break;
+          case 'TIMESTAMP': {
+            const timestamp = value?.[key];
+            if (typeof timestamp === "string") {
+              const date = Number(timestamp);
+              if (!isNaN(date)) {
+                newVal = date;
+              } else {
+                newVal = Date.parse(timestamp);
+              }
+            } else {
+              newVal = timestamp;
+            }
```

Comment:
to make sure that in the end it is still a number, let's check if `newVal` is `NaN` then let's throw an error